yt_dlp/YoutubeDL.py

```diff
@@ -29,6 +29,7 @@
 from string import ascii_letters
 from zipimport import zipimporter
 
+from .aes import compat_pycrypto_AES
```

Comment:
This risks circular import.

Reply:
How? The top level modules don't import YoutubeDL